install/stages/4-rpi.sh

```diff
@@ -65,3 +62,19 @@ EOF3'
 sudo chmod 755 /etc/systemd/system/ros.service
 sudo systemctl enable ros
 echo "sudo systemctl start ros"
+
+echo "[+] Adding Uart config on boot"
+sudo bash -c 'cat << EOF3 > /etc/systemd/system/uart.service
+[Unit]
+Description=Launch Uart config on boot.
+
+[Service]
+Type=simple
+ExecStart=/home/ubuntu/Poseidon/install/stages/rpi/uart_on_boot.sh
```

Comment:
We'd need to put the service scripts into somewhere more explicit than the installer. I'd suggest a services folder, but I'm open to suggestions.